deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# -------- Build Settings -------- #
+# directory to deploy
+BUILD_DIR="./dist"
+# directory on target server
+DEPLOY_DIR="./latest"
+# use to install dependencies without disruption
+TEMP_DIR="./staged"
+# server to connect to
+TARGET="hanbot@hanbot.elviswolcott.com"
+# file to run
+ENTRY="index.js"
+# name to give job (mostly for debugging on server)
+JOB_NAME="hanbot"
+
+# --------- Create .env ---------- #
+echo "\
+DISCORD_WELCOME_CHANNEL=${DISCORD_WELCOME_CHANNEL}
+DISCORD_ADMITTED_ROLE=${DISCORD_ADMITTED_ROLE}
+DISCORD_ACTIVE_ROLE=${DISCORD_ACTIVE_ROLE}
+DISCORD_TOKEN=${DISCORD_TOKEN}
+ZOOM_API_KEY=${ZOOM_API_KEY}
+ZOOM_API_SECRET=${ZOOM_API_SECRET}
+ZOOM_WEBHOOK_SCRET=${ZOOM_WEBHOOK_SECRET}
+ZOOM_MEETING_ID=${ZOOM_MEETING_ID}
+ZOOM_TIME_THRESHOLD=${ZOOM_TIME_THRESHOLD}
+ZOOM_TIME_ANNOUNCEMENT_CHANNEL=${ZOOM_TIME_ANNOUNCEMENT_CHANNEL}
+TRAVIS_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER}
+TRAVIS_COMMIT=${TRAVIS_COMMIT}
+" >> ".env"
+
+
+# --------- Deploy Steps --------- #
+# remove extra files
+find "${BUILD_DIR}" -name "*.map" -type f -delete
```

Comment:
Don't delete the source maps (files ending with `.map`), they're used to generate proper error messages based on source files instead of compiled output.

Reply:
True (this came from a project where source maps were being fed into Bugsnag)